### ISSUE TITLE
class used in a function without importing it first

### ISSUE THREAD

**Issue by shantanuo** [rushter/heamy#9]
I tried the "heamy" module as shown in this example and it works as expected.

https://github.com/rushter/heamy/blob/master/examples/walkthrough.ipynb

I expected the line 24 to fail as Sequential class is not imported anywhere in the script. mlp_model function should not complete without error, I guess. What am I missing?

**Comment by rushter** [comment]
You are right. I think I've removed it accidentally. That was a model from keras.

**Comment by shantanuo** [comment]
ok. But the point is that even if you have not imported that model from keras, the code runs flawlessly without any error. Does it means that the function **mlp_model** never really gets called?

**Comment by rushter** [comment]
Yes, because in the code, I didn't add to the pipeline. I just demonstrated how to use functions.
